API/api/repositories/event_repository.py

```
@@ -12,4 +12,7 @@ def get_event_by_id(self, event_id):
             return Event.objects.get(pk=event_id)
         except Event.DoesNotExist:
             return None
+        
+    def get_upcoming_events(self):
+        return Event.objects.filter(event_date__gte=date.today()).order_by('event_date')
```

Ruff (0.15.12)

[warning] `datetime.date.today()` used

(DTZ011)